src/qt/locale/bitcoin_pt_PT.ts

```diff
@@ -4200,303 +4204,303 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>The %s developers</source>
-        <translation type="unfinished"></translation>
+        <translation>Os desenvolvedores %s</translation>
     </message>
     <message>
         <source>Bitcoin Core and Dogecoin Core</source>
-        <translation type="unfinished"></translation>
+        <translation>Bitcoin Core e Dogecoin Core</translation>
     </message>
     <message>
         <source>-discardthreshold is set very high! This is the output amount that the wallet will discard (to fee) if it is smaller than this setting.</source>
-        <translation type="unfinished"></translation>
+        <translation>-discardthreshold está definido muito alto! Este é o valor de saída que a carteira descartará (taxa) se for menor que essa configuração.</translation>
```

Comment:
O limite de descarte...

I believe that discard threshold is separated not together as one word.